templates/ts-react-apollo-client/schema.graphql

```diff
@@ -0,0 +1,173 @@
+## NOTE: This schema was generated by Graphback and should not be changed manually
+
+"""Exposes a URL that specifies the behaviour of this scalar."""
+directive @specifiedBy(
+  """The URL that specifies the behaviour of this scalar."""
+  url: String!
+) on SCALAR
+
+""" @model """
+type Comment {
+  """ @id """
+  _id: ID!
+  text: String
+  description: String
+
+  """@manyToOne(field: 'comments', key: 'noteId')"""
+  note: Note
+}
+
+input CommentFilter {
+  _id: IDInput
```

Comment:
So this may not solve the problem for mongo's ObjectID?

Reply:
It actually works with the mongo and postgress. Despite being not the proper type application works on the runtime and compiles. Types here are mostly informative and they all map to strings anyway.

Reply:
I can't test this for now (away from my machine) so I trust you on this.

Reply:
Never trust me 🤣